src/index.tsx

```diff
@@ -826,6 +826,98 @@ export function useBandwidthEstimation() {
   return { estimation };
 }
 
+/**
+ * This hook provides access to current rtc statistics data.
+ */
+export function useRTCStatistics() {
```

Comment:
this hook looks like something that can be unit tested, there are are bit of calculations, wdyt?

Reply:
Mby in the follow up? This PR is already 600+ long